containerd/socket-tracer/containerd-socket-tracer.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: containerd-socket-tracer
+  namespace: default
+  labels:
+    k8s-app: containerd-socket-tracer
+spec:
```

Comment:
must ignore not-ready to be deployed first. There may be cases when pauload access CRI socket on bootstrap and never again